codeSnippets/snippets/tutorial-websockets-client/README.md

```diff
@@ -8,7 +8,7 @@ A sample Ktor project showing how to create a [WebSocket chat client](https://kt
 Before running this sample, start a [Ktor server](../tutorial-websockets-server) first:
```

Comment:
Looks like the folder was renamed. Can you update the location here as well?

Reply:
Should I change this line too?

https://github.com/ktorio/ktor-documentation/blob/9ec4bcabd2431d10fa1fb70d54b6745315c68e71/topics/client-create-websocket-chat.md?plain=1#L27

In this case, the link points to the file in version `2.3.4` in which the change was not applied. Should I change it to `main`?

Reply:
Yes, well spotted! You can replace `2.3.4` in the URL with `%ktor_version%`. It will then automatically point to the relevant branch.